pkg/resources/saml2_integration_acceptance_test.go

```diff
@@ -78,17 +78,17 @@ func TestAcc_Saml2Integration_basic(t *testing.T) {
 				ConfigVariables: m(issuer, string(sdk.Saml2SecurityIntegrationSaml2ProviderCustom), validUrl, cert, false, false),
 				Check: resource.ComposeAggregateTestCheckFunc(
 					resource.TestCheckResourceAttr("snowflake_saml2_integration.test", "name", id.Name()),
-					resource.TestCheckResourceAttr("snowflake_saml2_integration.test", "enabled", "unknown"),
+					resource.TestCheckResourceAttr("snowflake_saml2_integration.test", "enabled", "default"),
```

Comment:
we could just use the const we have, right?

Reply:
yeah, but there's strange conflict. We have our resource names needed for CheckDestroy under "resources" package and our main package for resources is also named "resources". Because I was unsure how do we want to resolve this, I just used plain values.

Reply:
They are already used with this problem solved (named `r` import)

Reply:
Addressed in https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2931